### PR TITLE
Added the COC For the Collab Community 

### PR DESCRIPTION
**Pull request by Abhinavkar**

### CODE REVIEW

**Comment by KendallDoesCoding** [comment]
@Adarsh-jaiss please review at the earliest.

**Comment by Abhinavkar** [comment]
@KendallDoesCoding @vikasganiga05  Review

**Review by KendallDoesCoding — CHANGES_REQUESTED** [review]
ignore this comment

**Review by KendallDoesCoding — CHANGES_REQUESTED** [review]
File name is required to be CODE_OF_CONDUCT.md to be recognized by GitHub but right now it's CodeOfConduct.md

fixed

**Review by KendallDoesCoding — COMMENTED** [review]
Which one did you pick?

Contributor Covenant
Recommended for projects of all sizes

Citizen Code Of Conduct
Suitable for large communities and events

**Comment by Abhinavkar** [comment]
> Which one did you pick?
> 
> Contributor Covenant Recommended for projects of all sizes
> 
> Citizen Code Of Conduct Suitable for large communities and events

1 st one

**Comment by KendallDoesCoding** [comment]
> > Which one did you pick?
> > Contributor Covenant Recommended for projects of all sizes
> > Citizen Code Of Conduct Suitable for large communities and events
> 
> 1 st one

Ok cool!

**Review by Abhinavkar — COMMENTED** [review]
I have reviewed it

File: .github/CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+will maintain confidentiality with regard to the reporter of an incident.
+Enforcement may result in an indefinite ban from all official Collab community communication
+channels, or other actions as deemed appropriate by the Collab community team.
+
+Collab community maintainers who do not follow or enforce the Code of Conduct in good
+faith may face temporary or permanent repercussions as determined by other
+members of the project's leadership.
+
+
+# Regards Collab community
```
Comment (KendallDoesCoding) [inline .github/CODE_OF_CONDUCT.md:70]:
```suggestion
**Regards**,
Collab-Community
```

**Review by Abhinavkar — COMMENTED** [review]
I think Its good to go !

File: .github/CODE_OF_CONDUCT.md (hunk 1 above)
Comment (KendallDoesCoding) [reply .github/CODE_OF_CONDUCT.md:70]: @Abhinavkar please commit this

**Comment by KendallDoesCoding** [comment]
> I think Its good to go !

Please commit my suggestion, then we'll wait for @Adarsh-jaiss 's feedback

**Review by KendallDoesCoding — APPROVED** [review]

**Comment by KendallDoesCoding** [comment]
Approved by @Adarsh-jaiss . Thanks for your contribution @Abhinavkar

**Pull request merged**